client/src/main/scripts/slipstream.bootstrap.py

```diff
@@ -316,18 +327,94 @@ def getCloudName():
         raise RuntimeError('%s environment variable is not defined.' % envCloud)
 
 
-def _getTargetScriptCommand(targetScript):
+def get_machine_executor_command(executor_name):
+    if _system_supports_initd():
+        return _setup_and_get_initd_service_start_command(executor_name)
+    else:
+        return _get_machine_executor_direct_startup_command(executor_name)
```

Comment:
It may be interesting to fall-back to `_get_machine_executor_direct_startup_command` if `_setup_and_get_initd_service_start_command` fails.